Package.swift

```diff
@@ -30,7 +30,7 @@ let package = Package(
     dependencies: [
         .package(
             url: "https://github.com/groue/GRDB.swift.git",
-            .upToNextMinor(from: "6.12.0")
+            .upToNextMinor(from: "6.26.0")
```

Comment:
This was necessary to address these warnings:

![Screenshot 2024-08-20 at 5 31 06 PM](https://github.com/user-attachments/assets/9e6866ca-d895-415a-a6e5-2bbe8a48c79f)

![image](https://github.com/user-attachments/assets/d08e64ff-3a37-4e7f-816c-0a1030a8a9fd)

They are coming from GRDB itself, which is strange, but may be due to Dejavu being included in the Examples app as a local package. 🤷🏼‍♂️ In any case, requiring a newer version of GRDB took care of it.

Reply:
Oh! These warnings are visible when building with Xcode 15.4, even when building the Dejavu package itself. Good to address them, but I'll pull them out of this PR and put them in a separate one.